Makefile

```diff
@@ -11,4 +11,4 @@ update:
 	docker-compose run --rm cli bash -c "composer update"
 
 test:
-	docker-compose run --rm cli bash -c "cd /data/tests; ./phpunit ."
+	docker-compose run --rm cli bash -c "cd /data/tests; ../vendor/bin/phpunit ."
```

Comment:
If you are going to change this, then why not delete the softlink too?

Reply:
That's strange. I have a softlink, but there's not such thing in the source. Nevermind. :)

Reply:
What softlink? I don't see one.